### ISSUE TITLE
Instantiate BTC pallets

### ISSUE THREAD

**Issue by kylezs** [chainflip-io/chainflip-backend#2887]
Blocked by the BTC cryptography tasks, since we'll need a lot of the types to use in the instantiation.

### Description

We will need to instantiate the BTC pallets (vaults, threshold sig, broadcast, chain tracking, ingress-egress) and include them in the runtime.

## BTC Vault rotations
There are currently 2 stages to vault rotations, keygen and activation, as on the `VaultRotator` trait. In BTC they would be as follows: 
- keygen = keygen
- activation = sending funds from old wallet to new wallet. this also involves deciding on a specific block number we consider the new key "active", this block number will be used as the boundary between old and new epoch on BTC.

There may need to be something more custom for rejuvenation, but this should be implemented as separate issue. This issue should be instantiating the pallets such that the baseline features of BTC will work.

The rest should be fairly obvious, plugging in types. Chain tracking can be implemented as a separate issue #2872 

### Relevant people

@dandanlen @martin-chainflip
